src/geometry/manifold/smooth_manifold_with_corners.lean

```diff
@@ -703,45 +701,242 @@ not `local_homeomorph` as the target is not open in `E` in general, but we can s
 as `local_equiv`.
 -/
 
+namespace local_homeomorph
+/-- Given a chart `f` on a manifold with corners, `f.extend` is the extended chart to the model
+vector space. -/
+@[simp, mfld_simps] def extend : local_equiv M E :=
+f.to_local_equiv ≫ I.to_local_equiv
+
+lemma extend_coe : ⇑(f.extend I) = I ∘ f := rfl
+
+lemma extend_coe_symm : ⇑(f.extend I).symm = f.symm ∘ I.symm := rfl
+
+lemma extend_source : (f.extend I).source = f.source :=
+by rw [extend, local_equiv.trans_source, I.source_eq, preimage_univ, inter_univ]
+
+lemma is_open_extend_source : is_open (f.extend I).source :=
+by { rw extend_source, exact f.open_source }
+
+lemma extend_target : (f.extend I).target =
+  I.symm ⁻¹' f.target ∩ range I :=
+by simp_rw [extend, local_equiv.trans_target, I.target_eq, I.to_local_equiv_coe_symm, inter_comm]
+
+lemma maps_to_extend (hs : s ⊆ f.source) :
+  maps_to (f.extend I) s ((f.extend I).symm ⁻¹' s ∩ range I) :=
+begin
+  rw [maps_to', extend_coe, extend_coe_symm, preimage_comp, ← I.image_eq, image_comp,
+    f.image_eq_target_inter_inv_preimage hs],
+  exact image_subset _ (inter_subset_right _ _)
+end
+
+lemma extend_source_mem_nhds {x : M} (h : x ∈ f.source) :
+  (f.extend I).source ∈ 𝓝 x :=
+(is_open_extend_source f I).mem_nhds $ by rwa f.extend_source I
+
+lemma extend_source_mem_nhds_within {x : M} (h : x ∈ f.source) :
+  (f.extend I).source ∈ 𝓝[s] x :=
+mem_nhds_within_of_mem_nhds $ extend_source_mem_nhds f I h
+
+lemma extend_continuous_on : continuous_on (f.extend I) (f.extend I).source :=
```

Comment:
```suggestion
lemma continuous_on_extend : continuous_on (f.extend I) (f.extend I).source :=
```
? To match the order of symbols in the lemma statement, in the same way we have `continuous_exp` in the library and not `exp_continuous`. Same thing for many lemmas below.